packages/gatsby-source-mongodb/src/mapping.js

```diff
@@ -1,4 +1,7 @@
-module.export = function(node, key, text, createNode) {
+const _ = require(`lodash`),
+  crypto = require(`crypto`)
+
+module.exports = function(node, key, text, createNode) {
```

Comment:
Sorry, thought I'd perhaps explained this in the past — when you map fields, you should add a "link" from the field to the new node like this https://github.com/gatsbyjs/gatsby/blob/9452fa61f8731937057e2e2636d2301c237cd5f0/packages/gatsby-source-contentful/src/normalize.js#L286